TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AutonomousRight.java

```diff
@@ -679,19 +677,14 @@ private void autonomousCore() {
             // left turn 90 degree facing to high junction
             imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
             Logging.log("Autonomous - imu angle before left turn: %.2f", imuAngles.firstAngle);
-            rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle), AUTO_ROTATE_POWER); // turn robot 90 degree to the left
+            rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle) - 135, AUTO_ROTATE_POWER); // turn robot 90 degree to the left
             Logging.log("Autonomous - imu angle after turn: %.2f", lastAngles.firstAngle);
 
-            // adjust for accuracy
-            imuAngles = imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES);
-            rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle), AUTO_ROTATE_POWER);
-            Logging.log("Autonomous - imu angle after correction: %.2f", lastAngles.firstAngle);
-
             waitSliderRun(); // make sure slider has been lifted
             Logging.log("Autonomous - slider is positioned to high junction.");
 
             // moving forward V to junction
-            robotRunToPosition(8.5, true); // adjust according to testing
+            robotRunToPosition(13.5, true); // adjust according to testing
 
```

Comment:
Add a waiting period here to avoid junction shading

Reply:
We must also update the rotation angle after the for{} loop and before moving to the parking lot. Like below:

rotate(-AngleUnit.DEGREES.normalize(imuAngles.firstAngle-45), AUTO_ROTATE_POWER);